.eslintrc.js

```diff
@@ -86,6 +86,15 @@ module.exports = {
           'error',
           { argsIgnorePattern: '^_' },
         ],
+
+        // These rules are deprecated, but we have an old config that enables it
+        '@typescript-eslint/camelcase': 'off',
+        '@typescript-eslint/ban-ts-ignore': 'off',
+
+        // These rules are actually disabled in @xpring-eng/eslint-config-base/loose at the moment
+        '@typescript-eslint/no-unsafe-call': 'off',
+        '@typescript-eslint/no-unsafe-member-access': 'off',
+        '@typescript-eslint/no-unsafe-assignment': 'off',
```

Comment:
These are rules that are now enabled by default in `@typescript-eslint >= 3.0.0`, but cause various warnings. I don't even have them enabled in the default `@xpring-eslint/loose` configuration yet (but might do before I tag 1.0).